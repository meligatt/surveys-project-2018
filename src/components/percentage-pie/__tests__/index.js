import React from 'react';
import PercentagePie from '../index';

describe('PercentagePie', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <PercentagePie />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
