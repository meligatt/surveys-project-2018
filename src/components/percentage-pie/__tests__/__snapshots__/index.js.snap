// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PercentagePie renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PercentagePie />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="percentage-pie__container"
>
        <h4>
                Participation rate as a percentage
        </h4>
        <div
                className="percentage-pie__percentage"
        >
                NaN
                %
        </div>
</div>,
      "className": "percentage-pie",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h4>
            Participation rate as a percentage
</h4>,
          <div
            className="percentage-pie__percentage"
>
            NaN
            %
</div>,
        ],
        "className": "percentage-pie__container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Participation rate as a percentage",
          },
          "ref": null,
          "rendered": "Participation rate as a percentage",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "NaN",
              "%",
            ],
            "className": "percentage-pie__percentage",
          },
          "ref": null,
          "rendered": Array [
            "NaN",
            "%",
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="percentage-pie__container"
>
          <h4>
                    Participation rate as a percentage
          </h4>
          <div
                    className="percentage-pie__percentage"
          >
                    NaN
                    %
          </div>
</div>,
        "className": "percentage-pie",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h4>
              Participation rate as a percentage
</h4>,
            <div
              className="percentage-pie__percentage"
>
              NaN
              %
</div>,
          ],
          "className": "percentage-pie__container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Participation rate as a percentage",
            },
            "ref": null,
            "rendered": "Participation rate as a percentage",
            "type": "h4",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "NaN",
                "%",
              ],
              "className": "percentage-pie__percentage",
            },
            "ref": null,
            "rendered": Array [
              "NaN",
              "%",
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
